bot/exts/evergreen/cowsay.py

```diff
@@ -0,0 +1,62 @@
+import random
+
+from cowsay import char_names, get_output_string
+from discord import Embed
+from discord.ext import commands
+
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+
+class Cowsay(commands.Cog):
+    """Cog for the cowsay command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(
+        aliases=["cow"],
+        help=f"""
+        Generates a cowsay string and sends it. The available characters for cowsay are:
+
+        {', '.join(char_names)}"""
+    )
+    async def cowsay(self, ctx: commands.Context, character: str = "Cow", *, text: str = None) -> None:
+        """Builds a cowsay string and sends it to Discord."""
+        if not text:
+            text = f"I'm a {character.lower()}"
+        text = text.lower()
```

Comment:
I forgot to mention this, but why are we lowering the text here?

Reply:
@ToxicKidz all character names need to be lowered otherwise you get an error, i can stop lowering the text if you want.

Reply:
I'm talking about the text, not the characters, so yes

Reply:
i removed this, will commit soon